joomla/joomla-cms
# [com_banners] tracks view: searchtools ordering and bug corrections

#### Description

This PR corrects bugs in the com_banners tracks views searchtools and matches the searchtools layout with other views.
###### Before PR

![image](https://cloud.githubusercontent.com/assets/9630530/14584394/7dd792fe-043c-11e6-96d9-d4aefe36178d.png)
###### After PR

![image](https://cloud.githubusercontent.com/assets/9630530/14584390/4f114da2-043c-11e6-9db0-a9944b8c2d5e.png)
#### How to test
1. Use latest staging
2. Create banners clients, then create some banners and associate with those clients
3. Confirm you have some banner tracks for the tests (see note below)
4. Go to Components -> Banners -> Tracks
5. Check there are several bugs in the searchtools
6. Apply patch
7. Check the search, ordering select box and the filters are working fine.
8. Test desktop, tablet and mobile view (ex: reduce browser width).

Note: If you don't have tracks records you can generate some in the database (`#__banners_tracks` table), example:

``` sql
INSERT INTO `<prefix>_banner_tracks` (`track_date`, `track_type`, `banner_id`, `count`) VALUES ('2016-04-11 00:00:00', '1', '1', '5');
INSERT INTO `<prefix>_banner_tracks` (`track_date`, `track_type`, `banner_id`, `count`) VALUES ('2016-01-11 00:00:00', '2', '1', '7');
INSERT INTO `<prefix>_banner_tracks` (`track_date`, `track_type`, `banner_id`, `count`) VALUES ('2015-04-11 00:00:00', '2', '2', '9');
INSERT INTO `<prefix>_banner_tracks` (`track_date`, `track_type`, `banner_id`, `count`) VALUES ('2016-02-11 00:00:00', '2', '2', '12');
```
